devel/ci-cluster.sh

```diff
@@ -29,11 +29,6 @@ source "${SCRIPT_ROOT}/lib/lib.sh"
 # Configure PATH to use bazel provided e2e tools
 setup_tools
 
-export SERVICE_IP_PREFIX="10.0.0"
-if [[ "$IS_OPENSHIFT" == "true" ]] ; then
-  export SERVICE_IP_PREFIX="172.30.0"
-fi
```

Comment:
Is this because Kind no longer uses this fixed network range?

Reply:
With the latest kind it seemed like the ranges were different depending on what version of Kubernetes is being deployed, with the 1.23 it was now `10.96.0`. (Even if they weren't different, I think it probably makes more sense to check the range in code as that way we won't have mystery errors if the ranges change)